compose.yaml

```diff
@@ -92,7 +92,25 @@ services:
       - crawl_data:/tmp/crawl_data
     command: /bin/sh -c 'echo size-before && du -s /tmp/* && rm -rf /tmp/opensearch_data/* /tmp/crawl_data/* && echo size-after && du -s /tmp/*'
     
+  jupyter:
+    profiles: ['jupyter']
+    image: arynai/jupyter:${JUPYTER_VERSION}
```

Comment:
Isn't there a public version that we can point to?

Reply:
I don't understand the question. That is public.
https://hub.docker.com/repository/docker/arynai/jupyter/general